src/pp.js

```diff
@@ -334,9 +334,14 @@ function print(w, doc) {
               }
 
             case MODE_BREAK:
-              if (out.length > 0) {
+              if (out.length > 0 && doc.literal !== true) {
```

Comment:
Do you mind moving this into the `else` block below, then you can do `if(out.length > 0) ...` in there since it's already known to not be a literal line?

Would make the code flow a little easier to read since we want to do this only for non-literal lines